docs/website/root/manual/developer-docs/nodes/mithril-client-library.md

```diff
@@ -49,56 +48,100 @@ Mithril Client Library can be used by Rust developers to use the Mithril Network
 
 ## Installation
 
-In your project crate, use `cargo` to add [mithril-client](https://crates.io/crates/mithril-client) as a dependency:
+:::caution
+
+Mithril Client library has not yet been published on [crates.io](https://crates.io/), so you won't be able to follow the procedure below. The crate will be published soon.
```

Comment:
Can you add an alternative option with a git path in the dependency configuration in the `Cargo.toml` file?